packages/test/test-end-to-end-tests/src/test/sharedTreeEndToEndTests.spec.ts

```diff
@@ -0,0 +1,461 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+
+import { describeCompat } from "@fluid-private/test-version-utils";
+import type { IContainer } from "@fluidframework/container-definitions/internal";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { ITestObjectProvider } from "@fluidframework/test-utils/internal";
+import {
+	ITree,
+	SchemaFactory,
+	TreeViewConfiguration,
+	type TreeView,
+} from "@fluidframework/tree";
+import {
+	asTreeViewAlpha,
+	SharedTree,
+	type Revertible,
+	type TreeViewAlpha,
+} from "@fluidframework/tree/internal";
+
+describeCompat("SharedTree", "NoCompat", (getTestObjectProvider, apis) => {
```

Comment:
The doc comment on describeCompat says:

>  There are three compatVersion options to generate different combinations, depending of the need of the tests:
>  * `FullCompat`: generate test variants with compat combinations that varies the version for all layers.
>  * `LoaderCompat`: generate test variants with compat combinations that only varies the loader version.
>  * Specific version (String) : specify a minimum compat version (e.g. "2.0.0-rc.1.0.0") which will be the minimum version a
>  * test suite will test against. This should be equal to the value of pkgVersion at the time you're writing the new test suite.

"NoCompat" is not an allowed string based on that. Looks like the docs on describeCompat are just wrong?